### DISCUSSION TITLE
송인식 교수님 오픈소스 수업에서 유일하게 얻어간 것 (리주성은 꼭 봐야한다.)

### DISCUSSION THREAD

**Discussion by ghdcksgml1** [ghdcksgml1/Algorithm_Study#230] [자유게시판]
# 📘 갓갓 사이트

### @jusung-c @jusung-c @jusung-c @jusung-c @jusung-c @jusung-c @jusung-c

<img width="1252" alt="스크린샷 2022-04-29 오전 11 48 41" src="https://user-images.githubusercontent.com/79779676/165878098-e7da7b66-ce41-4ff8-a061-737d4cc6bdc9.png">

### @jusung-c @jusung-c @jusung-c @jusung-c @jusung-c @jusung-c @jusung-c

이 사이트를 알고나서 CLI와 깃의 네트워크 플로우를 이해할 수 있었습니다.

엄청 쉬우니까 2단계까지 꼭 여러번해보세연~

(물론, 송인식 교수님이 소개해주셨지만, 소개만해줌 ㅋㅋㅋㅋㅋㅋㅋㅋ)

링크 : https://learngitbranching.js.org/?locale=ko

**Comment by jusung-c** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ꿀팁 감사요ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ
